src/main/java/nextstep/sessions/domain/SessionStudents.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.sessions.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class SessionStudents {
+
+    private Set<SessionStudent> students = new HashSet<>();
+
+    public SessionStudents() {
+        this.students = new HashSet<>();
+    }
+
+    public SessionStudents(List<SessionStudent> students) {
+        for (int i = 0; i < students.size() - 1; i++) {
+            checkStudent(students.get(i), students.get(i+1));
```

Comment:
`i`와 `+` 사이, `+`와 `1` 사이에 공백이 누락됐네요.
```suggestion
            checkStudent(students.get(i), students.get(i + 1));
```
IDE의 자동 줄맞춤 단축키를 사용하시거나, 파일을 저장할 때마다 자동으로 줄맞춤이 되도록 설정해보시면 좋겠습니다.
intellij를 사용하신다면 Preferences > Tools > Actions on Save 에서 자바 코드도 Reformat Code, Optimize Imports를 설정해서 수정하는 코드를 자동 줄맞춤, 임포트 최적화를 실행하도록 하실 수 있습니다. :)